src/features/connect/ConnectProfileSelectScreen.tsx

```diff
@@ -57,6 +57,7 @@ export const ConnectProfileSelectScreen = ({ navigation, route }: Props) => {
    * to generate grants for each selected DID.
    */
   const connect = async () => {
+    console.log('Fetching request from Auth Request URI:', request_uri);
```

Comment:
Yes, intentional, pending response to other comment.

Reply:
I will remove the logs simply because I do not want this PR blocked on logging discussion, however we need to reach consensus on logging principles, what are yours?

My view is that we log far less than we should across apps and services: we should log more, not less in general.

Reply:
It is now clear to me what you want is zero debug/informational logs, I see some valid points you are making, but disagree with the overall principle.

Since you are the DRI, you get to make the call.